docs/reference/supporting_resources.md

```diff
@@ -2,4 +2,26 @@
 title: Supporting Resources
 ---
 
-This is a stub
+Great Expectations requires a python compute environment and access to data, either locally or through a database or distributed cluster. In addition, developing with great expectations relies heavily on tools in the Python engineering ecosystem: pip, virtual environments, and jupyter notebooks. We also assume some level of familiarity with git and version control.
+
+See the links below for good, practical tutorials for these tools.
+
+#### pip
+[https://pip.pypa.io/en/stable/](https://pip.pypa.io/en/stable/)
+[https://www.datacamp.com/community/tutorials/pip-python-package-manager](https://www.datacamp.com/community/tutorials/pip-python-package-manager)
+
+#### virtual environments
+[https://virtualenv.pypa.io/en/latest/](https://virtualenv.pypa.io/en/latest/)
+[https://python-guide-cn.readthedocs.io/en/latest/dev/virtualenvs.html](https://python-guide-cn.readthedocs.io/en/latest/dev/virtualenvs.html)
+[https://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/](https://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/)
+  
+#### jupyter notebooks and jupyter lab
+[https://jupyter.org/](https://jupyter.org/)
+[https://jupyterlab.readthedocs.io/en/stable/](https://jupyterlab.readthedocs.io/en/stable/)
+[https://towardsdatascience.com/jupyter-lab-evolution-of-the-jupyter-notebook-5297cacde6b](https://towardsdatascience.com/jupyter-lab-evolution-of-the-jupyter-notebook-5297cacde6b)
+
+#### git
+[https://git-scm.com/](https://git-scm.com/)
+[https://reference.github.com/](https://reference.github.com/)
+https://www.atlassian.com/git/tutorials](https://www.atlassian.com/git/tutorials)
+
```

Comment:
These are all really great resources, thank you for adding them here! I've made a small suggestion to standardize the capitalization of some of the technologies so we are consistent throughout our docs. Very low impact but since our org is very particular about terminology, I think bringing that same attention to detail with spelling/capitalization makes sense.


```suggestion
Great Expectations requires a Python compute environment and access to data, either locally or through a database or distributed cluster. In addition, developing with great expectations relies heavily on tools in the Python engineering ecosystem: pip, virtual environments, and Jupyter Notebooks. We also assume some level of familiarity with Git and version control.

See the links below for good, practical tutorials for these tools.

#### pip
[https://pip.pypa.io/en/stable/](https://pip.pypa.io/en/stable/)
[https://www.datacamp.com/community/tutorials/pip-python-package-manager](https://www.datacamp.com/community/tutorials/pip-python-package-manager)

#### Virtual Environments
[https://virtualenv.pypa.io/en/latest/](https://virtualenv.pypa.io/en/latest/)
[https://python-guide-cn.readthedocs.io/en/latest/dev/virtualenvs.html](https://python-guide-cn.readthedocs.io/en/latest/dev/virtualenvs.html)
[https://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/](https://www.dabapps.com/blog/introduction-to-pip-and-virtualenv-python/)
  
#### Jupyter Notebooks and Jupyter Lab
[https://jupyter.org/](https://jupyter.org/)
[https://jupyterlab.readthedocs.io/en/stable/](https://jupyterlab.readthedocs.io/en/stable/)
[https://towardsdatascience.com/jupyter-lab-evolution-of-the-jupyter-notebook-5297cacde6b](https://towardsdatascience.com/jupyter-lab-evolution-of-the-jupyter-notebook-5297cacde6b)

#### Git
[https://git-scm.com/](https://git-scm.com/)
[https://reference.github.com/](https://reference.github.com/)
https://www.atlassian.com/git/tutorials](https://www.atlassian.com/git/tutorials)

```